scipy/spatial/distance.py

```diff
@@ -144,6 +146,67 @@ def _validate_vector(u, dtype=None):
         raise ValueError("Input vector should be 1-D.")
     return u
 
+def directed_hausdorff(u, v):
+    """
+    Computes the directed Hausdorff distance between N-dimensional
+    point sets using a Euclidean metric.
```

Comment:
Can you make this fit on one line followed by a blank line? First line gets extracted to summary table in the html docs, so needs to be short enough.